client/src/Components/Properties/PropSingle/Property.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from "react";
+import { FlexColumn, FlexRow } from "custom-components";
+import { Checkbox, Button } from "semantic-ui-react";
+
+class Property extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: "",
+      address: "",
+      city: "",
+      state: "",
+      zip: "",
+      price: 0,
+      cleaningFee: 0,
+      maxGuests: 0,
+      employee: "",
+      imageLoc: "",
+      loading: false,
+      error: null
+    };
+  }
+  componentDidMount() {
+    this.props.getProperty();
+  }
+
+  componentWillReceiveProps(nextProps) {
```

Comment:
You actually don't need to set these on state. You can just pull them from props.